CHANGES/8252.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed content not being read when an upgrade request was not supported with the pure Python implementation.
```

Comment:
@bdraco it's best not to put a trailing period at the end of the sentence if the byline is right after — RST will render it on the same line. If you want a separate line, add an extra empty separator line in between. Look how it's rendered now: https://docs.aiohttp.org/en/latest/changes.html#bug-fixes.

Also, I suggest not using any period after the byline if it's on a dedicated line as opposed to being at the end of a sentence.

Examples:
* https://yarl.aio-libs.org/en/latest/changes/#id5 — look at the last entry — here I have a byline at the end of the first sentence/paragraph. So the sentence itself doesn't have a period before the em dash but it ends with one right after the byline. It's followed by a separate paragraph of additional text.
* https://multidict.aio-libs.org/en/latest/changes/#id3 — most entries here have bylines on separate lines, I only missed to drop a trailing period from one, but others are nice.